src/Samhammer.TimedHostedService/Samhammer.TimedHostedService.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@LeonSnajdr please update stylecop to 1.2 beta

Reply:
@LeonSnajdr please update project package definitions to include Readme.md and remove version property.
see other Samhammer projects for example.

Reply:
@LeonSnajdr please also add description to package definition and to github project.